crates/bevy_asset/Cargo.toml

```
@@ -30,7 +30,7 @@ anyhow = "1.0.4"
 thiserror = "1.0"
 downcast-rs = "1.2.0"
 fastrand = "1.7.0"
-notify = { version = "=5.0.0-pre.15", optional = true }
+notify = { version = "=5.0.0", optional = true }
```

```suggestion
notify = { version = "5.0.0", optional = true }
```
The only reason we used `=` is because the pre-releases had breaking changes, yet cargo considers them semver compatible.

---

Yup, that makes sense